deploy/internal/nsfs-pvc-cr.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: default
+spec:
+  storageClassName: default
```

Comment:
we can remove this

Reply:
Just to be clear - You are suggesting this yaml to be look like this - 

apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: noobaa-default-nsfs-pvc
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 30Gi

Reply:
done.